Formula/arcs.rb

```diff
@@ -40,9 +41,17 @@ def install
       "--prefix=#{prefix}",
       "--with-boost=#{Formula["boost"].opt_include}"
     system "make", "install"
+    libexec_src = Pathname.new("#{libexec}/bin/src")
+    libexec_src.install "src/long-to-linked-pe"
+    libexec_bin = Pathname.new("#{libexec}/bin/Examples")
+    libexec_bin.install "Examples/makeTSVfile.py"
+    libexec_bin.install "Examples/arcs-make"
+    (bin/"arcs-make").write_env_script libexec/"bin/Examples/arcs-make", PYTHONPATH: ENV["PYTHONPATH"]
```

Comment:
```suggestion
    libexec.install "src/long-to-linked-pe"
    libexec.install "Examples/makeTSVfile.py"
    libexec.install "Examples/arcs-make"
    (bin/"arcs-make").write_env_script libexec/"arcs-make", PYTHONPATH: ENV["PYTHONPATH"]
```
I'd suggest installing these three files in `libexec`. The directory names `libexec/bin/src` and `libexec/bin/Examples` seem unnecessarily long and clunky.

Reply:
I know it might not be as clean, but they are formatted like that because that's where the `arcs-make` makefile expects them to be (relative to where it is)... We could change for a future release, but that's how it's organized right now.

Reply:
Add these scripts to `dist_bin_SCRIPTS` in `Makefile.am`. See for example https://github.com/bcgsc/abyss/blob/da2d0da486443da9b3d1e107fd495e274b709e90/bin/Makefile.am#L8. It'll make this Homebrew formula way easier to maintain if `make install` installs all the required scripts to `bin`.

Reply:
Please add an `arcs-*` prefix to these executables if they're being installed to `bin`.